Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -53,11 +53,12 @@ private void OnClosing(object sender, CancelEventArgs e)
         private void OnInitialized(object sender, EventArgs e)
         {
             // show notify icon when flowlauncher is hided
```

Comment:
this comment goes with the method right?

Reply:
also while you are at it please change word from 'hided' to 'hidden' please

Reply:
Sure.